pkg/messaging/bots.go

```diff
@@ -0,0 +1,12 @@
+package messaging
+
+type Bots []Bot
+
+func NewBots(bots ...Bot) Bots {
+	return bots
+}
```

Comment:
I think this function is unnecessary. You can easily convert `[]Bot` slice to `Bot` type.